cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -66,7 +66,7 @@ while read -r bucket; do
   if [[ ${gcs_bucket_age_cutoff_seconds} -ge $( date --date="${creationTime}" '+%s' ) ]]; then
     buckets+=("${bucket}")
   fi
-done <<< $( gsutil -m ls -L -b 'gs://ci-op-*' | grep -Po "(gs:[^ ]+)|(?<=Time created:).*" )
+done <<< $( gsutil -m ls -p 'openshift-gce-devel-ci' -L -b 'gs://ci-op-*' | grep -Po "(gs:[^ ]+)|(?<=Time created:).*" )
```

Comment:
why would this script have access to any other projects?

Reply:
although I guess we are hard-coding the project name in existing master stuff.  Did you want this on the following `gsutil -m rm ...` call too?

Reply:
I don't know why it does, but it does. `gsutil rm` does not take a `-p` /shrug